SPECS-SIGNED/grub2-efi-binary-signed/grub2-efi-binary-signed.spec

```diff
@@ -58,6 +61,8 @@ specifically created for installing on %{buildarch} systems
 Summary:        GRand Unified Bootloader
 Group:          Applications/System
 Requires:       grub2-tools-minimal = %{version}-%{release}
+Recommends:     shim-x64 >= 15.8-1
```

Comment:
Curious - why the recommendation for shim-x64?

Reply:
it may make more sense if you review the commits individually, e.g. https://github.com/microsoft/azurelinux/commit/6d6b59851daaa43d6e14c8f70b70d6ef4fd30ddd